shell_sort/ShellSort.java

```diff
@@ -0,0 +1,40 @@
+/*Shell Short function 
+Worst case time complexity = O(n^2)
+ Best case complexity = O(nlog(n))
+n is input size*/
```

Comment:
@jsroyal  the comment block is related to `shellSort()`.
```
/*
Shell Short function 
Worst case time complexity = O(n^2)
Best case complexity = O(nlog(n))
n is input size
*/
```
this block belongs to line 6. Also look at the  @[Javadoc](https://google.github.io/styleguide/javaguide.html#s7-javadoc) in Google Java Style Guide.